src/main/java/ai/rev/helpers/ErrorInterceptor.java

```diff
@@ -31,6 +32,8 @@ public Response intercept(Interceptor.Chain chain) throws IOException {
           throw new AuthorizationException(errorResponse);
         case 400:
           throw new InvalidParameterException(errorResponse);
+        case 403:
+          throw new ForbiddenRequestException(errorResponse);
```

Comment:
is that possible to create a test for this exception?

Reply:
added it to the the test suite